gtfs-realtime/spec/en/trip-modifications.md

```diff
@@ -0,0 +1,65 @@
+
+* A TripUpdate SHOULD be provided using a `ModifiedTripSelector` inside the TripUpdate's `TripDescriptor`. 
+    * When the TripUpdate refers to the replacement trip, the consumer should behave as if the static GTFS would have been modified with the TripModifications (e.g. `arrival_time`, `departure_time`, `stop_sequence`, `stop_id` on replacement stops)
+    * When providing a `ModifiedTripSelector`, the other fields of the `ScheduleRelationship` SHOULD be left empty, to avoid confusion by consumer that aren't looking for the `ModifiedTripSelector` value. 
+    * Providing a TripUpdate with the a `ModifiedTripSelector` is the only way to create predictions at replacement stops
+* If no such TripUpdate is found, TripUpdates for the original `trip_id` will apply to the modified trip. 
+    * In this case, the static GTFS information used should be from the static GTFS before any TripModifications applied. 
+    * Real time can be available to the common stops between the previous trip and the new modified trip, no ETA is available at the replacement stops.
+
+##  Modification
+
+A `Modification` message replaces a span of n stop times (`num_stops_replaced`) from each affected trip starting at `start_stop_sequence`. The spans of the modifications MUST not overlap. Spans may not be contiguous; in this case the two modifications MUST be merged into one.  These stop times are replaced with a new stop time for each replacement stop described by `replacement_stops`.
+
+The sequence of `replacement_stops` may be of arbitrary length. For example, 3 stops could be replaced by 2, 4, or 0 stops as the situation may require.
+
+![](images/trip_modification.png)
+
+_An example showing the effect of a modification on a particular trip. This modification may also be applied to several other trips._
+
+![](images/propagated_delay.png)
+
+_Propagated detour delays affect all stops following the end of a modification. If a trip has multiple modifications, the delays are accumulated._
+
+## ReplacementStop
+
+Each `ReplacementStop` message defines a stop that will now be visited by the trip, and optionally specifies the estimated travel time to the stop. The `ReplacementStop` message is used to construct the scheduled `stop_time` for the stop.
+
+When `travel_time_to_stop` is specified, the `arrival_time` is calculated from a reference stop in the original trip, plus the offset in `travel_time_to_stop`. Otherwise, the `arrival_time` can be be interpolated based on the total duration of the modification in the original trip.
+
+The `departure_time` always equals the `arrival_time`.
+
+The optional fields of [`stop_times.txt`](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stop_timestxt) in the (CSV) GTFS specification are all set to their default values.
+
+![](images/first_stop_reference.png)
+
+_If a modification affects the first stop of the trip, that stop also serves as the reference stop of the modification._
+
+## Shape
```

Comment:
nit: I don't think there is value in including the Shape and Stop text here if they are going to replicate what's already specified in `[reference.md]`